src/js/common/schemaform/FormSaved.jsx

```diff
@@ -39,18 +39,22 @@ class FormSaved extends React.Component {
   render() {
     const { profile } = this.props.user;
     const lastSavedDate = this.props.lastSavedDate;
-    const prefillAvailable = !!(profile && profile.prefillsAvailable.includes(this.props.formId));
+    const formId = this.props.formId;
+    const prefillAvailable = !!(profile && profile.prefillsAvailable.includes(formId));
     const verifiedAccountType = 3;// verified ID.me accounts are type 3
     const notVerified = profile.accountType !== verifiedAccountType;
     const { success } = this.props.route.formConfig.savedFormMessages || {};
-    const expirationDate = moment.unix(this.props.expirationDate);
+    const expirationDate = moment.unix(this.props.expirationDate).format('D/M/YYYY');
```

Comment:
I think the month and day are swapped here.

Reply:
Good catch, changed.